### PR TITLE
added the isCompatibleWithInstalledVersion

### PR DESCRIPTION
**Pull request by MalfuncEddie**
Small fix to add the isCompatibleWithInstalledVersion info to the output.

https://javadoc.jenkins.io/hudson/model/UpdateSite.Plugin.html#isCompatibleWithInstalledVersion--

### CODE REVIEW

File: scripts/groovy/gather_jenkins_facts.groovy [hunk 1]
```diff
@@ -52,10 +52,14 @@ Map getPluginInfo(List<PluginWrapper> plugins) {
   Map ret = [:]
   for (PluginWrapper plugin in plugins) {
     UpdateSite.Plugin updateInfo = plugin.getUpdateInfo()
+    if (updateInfo != null) {
+        isCompatibleWithInstalledVersion = updateInfo.isCompatibleWithInstalledVersion()
```
Comment (tobias-richter) [inline scripts/groovy/gather_jenkins_facts.groovy:57]: I think we should declare isCompatibleWithInstalledVersion properly as boolean and initialize it with a default value of false for the case when `updateInfo` is `null`

```
Boolean isCompatibleWithInstalledVersion = false
if (updateInfo != null) {
  isCompatibleWithInstalledVersion = updateInfo.isCompatibleWithInstalledVersion()
}
```

**Review by tobias-richter — CHANGES_REQUESTED** [review]
@MalfuncEddie thanks for the contribution. Only one small item. When this is solved we can merge and I will release a new version.

File: scripts/groovy/gather_jenkins_facts.groovy (hunk 1 above)
Comment (MalfuncEddie) [reply scripts/groovy/gather_jenkins_facts.groovy:57]: added 
Boolean isCompatibleWithInstalledVersion = false

Comment (tobias-richter) [reply scripts/groovy/gather_jenkins_facts.groovy:57]: thanks!

**Review by tobias-richter — APPROVED** [review]

**Review by tobias-richter — APPROVED** [review]

**Pull request merged**